### PR TITLE
Feature groups

### PR DESCRIPTION
**Pull request by icanberk**

### CODE REVIEW

File: app/controllers/assessment/autograde.rb [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
-  # 
-  # autogradeAfterHandin - submits an autograding job to Tango when
-  # the students submit their work.
-  #
-  def autogradeAfterHandin(submission)
+  ##
+  # autogradeSubmissions - submits an autograding job to Tango.
+  # Called by assessments#handin, submissions#regrade and submissions#regradeAll
+  # returns the job status returned by sendJob
+  def autogradeSubmissions(course, assessment, submissions)
```
Comment (icanberk) [inline app/controllers/assessment/autograde.rb]: I don't understand. Why are we passing multiple submissions here but only using submissions[0] throughout?

**Comment by dlbucci** [comment]
Wait a minute, why is this a pull request?  This code is not done.

**Comment by icanberk** [comment]
I know. It just let's us discuss stuff.

**Comment by dlbucci** [comment]
Can't you do a pull request to develop instead?

**Comment by icanberk** [comment]
ugh my bad.

**Pull request closed without merging**